holidays/countries/guyana.py

```
@@ -0,0 +1,279 @@
+
+    References:
+        * [Public Holidays Amendment Act 1967](https://web.archive.org/web/20241125073357/https://parliament.gov.gy/documents/bills/11264-bill_6_of_1967_public_holidays.pdf)
+        * [Public Holidays Amendment Act 1969](https://web.archive.org/web/20250424211718/https://parliament.gov.gy/documents/bills/11891-bill_26_of_1969_1.pdf)
+        * [Public Holidays Act Consolidated as of 2012](https://web.archive.org/web/20250608202320/https://mola.gov.gy/laws/Volume%206%20Cap.%2018.01%20-%2023.011696968337.pdf)
+
+    The holidays below are not part of the latest Public Holidays Act but have been notified
+    every year as a holiday by Gazette notifications for many years. The earliest and latest
+    available notifications for each are linked below.
+
+    * Independence Day:
+        * [Extraordinary Gazette 14 May 2016](https://web.archive.org/web/20250210091920/https://officialgazette.gov.gy/images/gazettes-files/Extraordinary-gazette_14may161.pdf)
```

Pylint (3.3.7)

[convention] Line too long (177/100)

(C0301)